profile/rwahs.xml

```diff
@@ -7849,7 +7851,7 @@
             <setting name="remove_first_items"><![CDATA[1]]></setting>
             <setting name="hierarchy_order"><![CDATA[ASC]]></setting>
             <setting name="hierarchy_limit"><![CDATA[0]]></setting>
-            <setting name="hierarchical_delimiter"><![CDATA[ &rArr; ]]></setting>
+            <setting name="hierarchical_delimiter"><![CDATA[ => ]]></setting>
```

Comment:
The dompdf library shipped with CA apparently only supports some *special* characters (*special special*?). As this report is specifically a printed one, this makes the report fairly legible.